core/services/postgres/transaction.go

```diff
@@ -89,3 +91,75 @@ func DBWithDefaultContext(db *gorm.DB, fc func(db *gorm.DB) error) error {
 	defer cancel()
 	return fc(db.WithContext(ctx))
 }
+
+func WrapDbWithSqlx(rdb *sql.DB) *sqlx.DB {
+	db := sqlx.NewDb(rdb, "postgres")
+	db.MapperFunc(CamelToSnakeASCII)
+	return db
+}
+
+func SqlxTransaction(ctx context.Context, rdb *sql.DB, fc func(tx *sqlx.Tx) error, txOpts ...sql.TxOptions) (err error) {
```

Comment:
sqlx doesnt provide this?

Reply:
No, it just contains the raw methods. The wrapper small and similar to how the gorm one works